lib/internal/abort_controller.js

```diff
@@ -357,6 +362,24 @@ function transferableAbortController() {
   return AbortController[kMakeTransferable]();
 }
 
+/**
+ * @param {AbortSignal} signal
+ * @param {any} resource
+ * @returns {Promise<void>}
+ */
+async function aborted(signal, resource) {
+  if (signal === undefined) {
+    throw new ERR_INVALID_ARG_TYPE('signal', 'AbortSignal', signal);
```

Comment:
Have added this check since turns out changing the condition in `validateAbortSignal()` immediately results in the build breaking, so i assume its probably not a bug but would like a review here

Reply:
It seems like a common pattern across to the codebase to check for undefined and then validate the signal so i think its probably fine?